README.md

```diff
@@ -46,7 +46,8 @@ Finally, no additional third-party runtime dependencies  are required to use thi
 * Use **block-list** (`maskKeys`) or **allow-list** (`allowKeys`) for masking
 * Limited support for JSONPath masking in both  **block-list** (`maskJsonPaths`) and **allow-list** (`allowJsonPaths`)
   modes
-* Masking a valid JSON will always return a valid JSON
+* Masking a valid JSON will always produce a valid JSON.
+* If the input is not valid JSON, processing is guaranteed to complete, but the resulting masked output is undefined
```

Comment:
Nit: maybe make it one point, not two separate ones?

Reply:
@gavlyukovskiy I had that before but then my OCD kicked-in and I noticed that all previous ones don't have a period at the end so it would be weird. Now I see that I still added a period on the first one. I am severely disappointed in myself...